src/arch/riscv/RiscvFsWorkload.py

```diff
@@ -43,10 +44,12 @@ class RiscvBareMetal(Workload):
     bootloader = Param.String("File, that contains the bootloader code")
     bare_metal = Param.Bool(True, "Using Bare Metal Application?")
     reset_vect = Param.Addr(0x0, "Reset vector")
+    semihosting = Param.RiscvSemihosting(
+        NULL,
+        "Enable support for RISC-V semihosting by settings this parameter",
+    )
     auto_reset_vect = Param.Bool(
-        True,
-        "Use bootloader entry point as reset vector. "
-        "If `auto_reset_vect` is true, the `reset_vect` parameter is ignored.",
+        True, "Use bootloader entry point as reset vector. "
```

Comment:
This part should not be changed in the PR
```suggestion
        True,
        "Use bootloader entry point as reset vector. "
        "If `auto_reset_vect` is true, the `reset_vect` parameter is ignored.",
```